src/js/components/Button/Button.tsx

```diff
@@ -27,28 +29,32 @@ export type Props = {
 const Button: React.FC<ButtonAttributes & Props> = ({
   children,
   className,
+  forceSolidColor = false,
   icon,
   iconPosition = 'left',
   iconTheme,
   isBlock,
   isDisabled,
   isOutline,
+  loading = false,
   onClick,
   size,
   tag: Tag = 'button',
-  theme = 'primary',
+  theme = 'secondary',
   type = 'button',
   ...other
 }: Props) => {
   const classes = classNames(
     className,
     'btn',
-    `btn${isOutline ? '-outline' : ''}-${theme}`,
+    { [`btn${isOutline ? '-outline' : ''}-gradient-${theme}`]: !forceSolidColor },
+    `btn${isOutline ? '-outline' : ''}-${theme}`, // backup
```

Comment:
By default, buttons will be gradients but if we eventually use a button with the `success` theme that doesn't include a gradient we need to have some backup

Reply:
Do you have to explicitly pass `forceSolidColor` in order for this to use the backup? If so I wonder if it might be better to do something like have an array of available gradient themes, check to see if the specified theme is available as a gradient and if it's not use the backup. Something like:
```
const gradientThemes = ['success', 'info']
gradientThemes.includes(theme) ? <set the className to use the gradient> : <set it to use the solid colour>
```

Reply:
No, this is different
Let's say you want to create a button using the `secondary` colour. If you want the gradient then just set the `theme` to `secondary`. If you want to just have the basic purple (not gradient) use `forceSolidColor`.
If you want to create a button using the `success` colour.  Just set the `theme` to `success`. Since `success` does not have a gradient colour the first line (line 50) will still be added but will not add any style. The second one will be applied style.
The `forceSolidColor` is just to not use the gradient on theme that have one (Primary, secondary, tertiary-red, tertiary-green, tertiary-black, academy) 
If you want to understand better you can checkout the branch and play with it on storybook

Reply:
Sounds good to me.